prow/config.yaml

```diff
@@ -19,35 +19,35 @@ presubmits:
     always_run: true
     context: pull-cadvisor-e2e
     rerun_command: "@k8s-bot pull-cadvisor-e2e test this"
-    trigger: "@k8s-bot (pull-cadvisor-e2e )?test this"
+    trigger: "(^@k8s-bot (pull-cadvisor-e2e )?test this$|^/test( pull-cadvisor-e2e)?$)"
```

Comment:
it is for handling `\r\n`. IMO, we can have `\r?$` for `trigger`, but not `rerun_command`.